### DISCUSSION TITLE
Calendar Component

### DISCUSSION THREAD

**Discussion by neovia-ciro** [primefaces/community#2607] [PrimeReact] [answered]
Hi there, 

I am facing some trouble using this component when it is set to view="month". 
          <div className="col-2" key={index.toString()}>
            <Calendar 
              value={date} 
              /*onChange={(e) => setDate(e.value)}*/ 
              view="month" 
              dateFormat="mm/yy" 
              key={index.toString()} 
              locale="pt-BR"
              // transitionOptions={transitionOptions}
            />
          </div>
When I click, it appears a calendar with a good width, but after few ms, it is resized to the same component size (col-2).

I "solved" it increasing the timeout transitionOption, e.g. to 60.000ms. But,  it is not a solution. 

I also have tried panelStyle, and other options available in the documentation, but I could not achieve it. 

Can somebody help me, please?

Best regards,

Ciro

**Comment by melloware** [comment, marked as answer]
Already an open bug: https://github.com/primefaces/primereact/issues/5948

**Reply by neovia-ciro** [reply]
Thank you @melloware !

While it is not fixed, I used transitionOption setting timeout equal  0 so that user can't see this behavior. 

Best Regards,

Ciro
